docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
-recommended version for your operating system.
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1520719644125_Screen+Shot+2018-03-07+at+11.00.28+PM.png)
 
-## Install the "Hello World" starter
 
-Gatsby uses "starters" for starting new projects. Starters
-are partially built Gatsby sites that are pre-configured to help you get moving faster.
-There are several official starters and many others contributed from the Gatsby
-community! [See the Starters page for the full list](/docs/gatsby-starters/).
+Let’s take a look at the code we’ve just generated.
 
-To install a starter, first install Gatsby's command line program by running the following:
+## Gatsby’s project structure
+### ✋ Open up the code using your editor of choice.
 
-```sh
-npm install --global gatsby-cli
-```
+You should see something like this:
 
-Once that's installed, open a new terminal window and run the following commands to create a new Gatsby site in a directory called `tutorial-part-one` and then move to this new directory:
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1522382930687_Screen+Shot+2018-03-29+at+11.06.11+PM.png)
 
-```sh
-gatsby new tutorial-part-one https://github.com/gatsbyjs/gatsby-starter-hello-world
-cd tutorial-part-one
-```
 
-This command downloads the files for the starter and then installs the needed
-npm packages. It should take around 1.5-3 minutes to finish. It may look like
-nothing is happening at first; be patient!
+*Note: The editor shown here is Visual Studio. If you’re using a different editor, it will look a little different.*
 
-Now let's try running Gatsby!
+Below, we've listed each top-level files and directories you see in the default project directory. The main pieces we’ll be concerned with for this piece of the tutorial are the `src` directory and the `gatsby-config.js` file.
 
-Gatsby has a built-in development server. Let's start it up by running the
-following command:
+> 💡 For a high-level definiton of each piece, see the [page on Gatsby's project file structure](/docs/gatsby-project-structure/).
 
-```sh
-gatsby develop
-```
+1. `/node_modules`
+2. **`/src`**: This directory will contain all of the code related to what you will see on the front-end of your site (what you see in the browser), like your site header, or a page template. “Src” is a convention for “source code”.
+3. `.gitignore`
+4. `.prettierrc`
+5. `gatsby-browser.js`
+6. **`gatsby-config.js`**: This is the main configuration file for a Gatsby site. This is where you can specify information about your site (metadata) like the site title and description, which Gatsby plugins you’d like to include, etc. (Check out the [config docs](/docs/gatsby-config/) for more detail).
+7. `gatsby-node.js`
+8. `gatsby-ssr.js`
+9. `LICENSE`
+10. `package-lock.json`
+11. `package.json`
+12. `README.md`
+13. `yarn.lock`
```

Comment:
Nice! I like the focus on relevant parts of the starter and then the reference doc containing all the information.